lib/rubocop/rspec/language.rb

```diff
@@ -53,6 +53,25 @@ def node_pattern
         attr_reader :selectors
       end
 
+      # Built in RSpec selectors
+      module BuiltIn
+        LANGUAGE = {
+          'Hooks' => %i[
+            prepend_before
+            before
+            append_before
+            around
+            prepend_after
+            after
+            append_after
+          ]
+        }.freeze
```

Comment:
Do you think it's possible to move this to `config/default.yml`?

Reply:
I struggle with testing it, it works if `config` context is included in spec. I obviously don't want to do that manually for every spec file, so I tried to include it in `spec_helper`, but it didn't work for some reason.
I'll try to look into that again.

Reply:
Please feel free to include it for cop specs only:
```ruby
config.define_derived_metadata(type: :cop) do |metadata|
  metadata[:config] = true
end
```

If `cop: true` metadata is not set for cops, it's possible to set it as well:
```ruby
config.define_derived_metadata(file_path: %r{/spec/rubocop/cop/rspec/}) do |metadata|
  metadata[:type] = :cop
end
```

Reply:
@bquorning might have some ideas with stubbing `AllCops`.
I've recently discovered that in addition to `config` there's also a `all_cops_config` https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/rspec/shared_contexts.rb#L76
Might come handy?

Reply:
Yeah, I actually tried to override `all_cops_config`, but the thing I was struggling with turned out to be that metadata assignment. So seems to work now.